addons.conf

```diff
@@ -39,7 +39,7 @@ server {
         proxy_pass http://web/static/admin/;
     }
 
-    location ~ ^/api/(v3|v4|v4dev)/ {
+    location ~ ^/api/(v3|v4|v5)/ {
```

Comment:
actually, maybe just `location ~ ^/api/` ?

Reply:
too late, but I'll write another patch for that :)